modules/internal/Bytes.chpl

```diff
@@ -136,7 +136,8 @@ module Bytes {
 
     :returns: A new :mod:`bytes <Bytes>`
   */
-  inline proc createBytesWithBorrowedBuffer(x: c_string, length=x.size) {
+  inline proc createBytesWithBorrowedBuffer(x: c_string,
+                                            length=x.size) : bytes {
```

Comment:
Do these line breaks cause any changes to the docs or was it just > 80 characters?

Reply:
They don't affect the docs, they were just > 80 chars after adding the return type. You can view the docs if you download and extract them from the artifact here: https://github.com/chapel-lang/chapel/pull/18502/checks?check_run_id=3761565776. (Artifacts link is in the upper right as you view that page)